pysces/core2/InfixParser.py

```diff
@@ -50,7 +50,8 @@ class MyInfixLexer:
         'abs': 'abs',
         'exp': 'math.exp',
         'ln': 'math.log',
-        'log': 'math.log10',
+        'log': 'math.log',
```

Comment:
We have to be careful here as in MathML 2.0 log without a base is log10 while math.log is ln so I'm not convinced this is a good idea (http://www.zvon.org/comp/r/ref-MathML_2.html#Elements~log)